package.json

```diff
@@ -20,14 +20,16 @@
     "benchmark": "node ./bin/benchmark.js",
     "generate-changelog": "github-changes -o oncletom -r 'tld.js' -n ${npm_package_version} --only-pulls --use-commit-body",
     "lint": "jshint --config .jshintrc lib/**/*.js",
-    "postinstall": "node ./bin/postinstall.js",
+    "postinstall": "node ./bin/postinstall.js && npm run build && npm run build-min",
     "posttest": "npm run lint && npm run test-browser",
     "prepublish": "npm run update",
     "test": "nyc mocha -R dot -r env-test",
     "test-browser": "testling",
     "test-watch": "mocha -R dot -r env-test --watch",
     "update": "node ./bin/update.js",
-    "version": "npm run generate-changelog && git add CHANGELOG.md"
+    "version": "npm run generate-changelog && git add CHANGELOG.md",
+    "build": "browserify index.js --s tldjs > tld.js",
```

Comment:
Should it also run `npm run update` to fetch the latest version of publicsuffix list before bundling?

Reply:
yea running `npm run update` before bundling makes total sense, thanks for pointing out

what do you think about bundling in a postinstall script? i feel that can be omitted

Reply:
Maybe we should omit it from `postinstall`? I feel like keeping commands simple and predictable is a good thing, let's not do too much at once.